__tests__/chat.test.js

```diff
@@ -153,6 +151,30 @@ describe('Chat Methods', () => {
     })
   })
 
+  describe('Chat react', () => {
+    it('Allows a user to react to a valid message', async () => {
+      await alice1.chat.send(channel, message)
+      let messages = await alice1.chat.read(channel, {peek: true})
+      const messageToReactTo = messages[0]
+
+      await bob.chat.react(channel, messageToReactTo.id, ':poop:')
+      messages = await alice1.chat.read(channel, {peek: true})
+      const reaction = messages[0]
+      expect(reaction.id).toBe(messageToReactTo.id + 1)
+      expect(reaction.content.type).toBe('reaction')
+      expect(messages[1].reactions.reactions.poop).toHaveProperty(config.bots.bob1.username)
+    })
+
+    // it('Throws an error if given an invalid emoji', async () => {
```

Comment:
So it turns out you can give pretty much anything to the chat react method and it seems to work. This looks a little weird on the GUI when it isn't an emoji though:
<img width="240" alt="screen shot 2019-01-05 at 2 15 08 pm" src="https://user-images.githubusercontent.com/5677971/50729780-612e4e00-10f4-11e9-85e6-64600de521a8.png">
Is this something we should check for in the bot or the client?

Reply:
yeah it's a known bug/feature/whatever. someone jokingly used it to make a voting thing, like you could put "Yes" and "No" as 2 reactions and people could pick by clicking. it should be blocked in CORE not by our module, so you shouldn't have to do anything here.